client/state/posts/actions.js

```diff
@@ -68,3 +71,36 @@ export function requestSitePosts( siteId, query = {} ) {
 		} );
 	};
 }
+
+/**
+ * Triggers a network request to fetch a specific post from a site.
+ *
+ * @param  {Number}   siteId Site ID
+ * @param  {Number}   postId Post ID
+ * @return {Function}        Action thunk
+ */
+export function requestSitePost( siteId, postId ) {
+	return ( dispatch ) => {
+		dispatch( {
+			type: POST_REQUEST,
+			siteId,
+			postId
+		} );
+
+		return wpcom.site( siteId ).post( postId ).get().then( ( post ) => {
+			dispatch( receivePost( post ) );
+			dispatch( {
+				type: POST_REQUEST_SUCCESS,
+				siteId,
+				postId
+			} );
```

Comment:
>this should probably return the post in case someone consuming this wants to chain onto the Promise and use it. See http://jsbin.com/jufayefego/edit?js,console

Arguably, we shouldn't want to allow this, as the only consumption of data should be directly from the Redux state tree, not from an action thunk resolution. The only reason I opted to return the Promise in the first place is in facilitating test's awareness of the requests completing (as nock mocks aren't as immediate as you might expect). It may also come in handy for server-side rendering in knowing whether the response is ready to serve (based on the resolution of all initial actions), but again, not in specifically consuming any data result.